content/en/opentelemetry/agent/_index.md

```diff
@@ -7,9 +7,34 @@ private: true
   The Datadog Agent with embedded OpenTelemetry Collector is in private beta.
```

Comment:
Can we also add a link stating "request access" to take users to private beta sign up form? 
form link: https://www.datadoghq.com/private-beta/agent-with-embedded-opentelemetry-collector/ 
see attached screenshot as an example. 

![Screenshot 2024-08-28 at 2 27 40 PM](https://github.com/user-attachments/assets/1790991e-8b37-418d-90ad-1d165c930477)